browser/src/app.scss

```
@@ -26,6 +26,13 @@ $theme-colors-light: (
     display: flex;
 }
 
+// Note that we define this before selection highlights so that it will override
+// the styles of symbol that has both classes applied.
+.sourcegraph-document-highlight,
+.sourcegraph-document-highlight span {
```

Since we no longer override syntax highlighting colors
```suggestion
// Note that we define this before selection highlights so that it will override
// the styles of symbol that has both classes applied.
.sourcegraph-document-highlight {
```